pkgs/applications/networking/mailreaders/astroid/default.nix

```diff
@@ -1,24 +1,22 @@
-{ stdenv, fetchFromGitHub, scons, pkgconfig, gnome3, gmime3, webkitgtk24x-gtk3
-, libsass, notmuch, boost, wrapGAppsHook }:
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, gnome3, gmime3, webkitgtk24x-gtk3
```

Comment:
`webkitgtk24x-gtk3` (2.4.11) is marked as insecure because of several vulnerabilities and should not be used unless really necessary. Please try to change the dependency to `webkitgtk` (which is 2.20).

Reply:
This is an [ongoing upstream issue](https://github.com/astroidmail/astroid/issues/49), and as of yet hasn't been fixed. WebKit-GTK 2.4.11 and 2.20 have an incompatible API, so it is non-trivial to fix.

Reply:
Fine, so let's merge this anyway.